src/simulation/replanning/routing/alt_landmark_data.rs

```diff
@@ -0,0 +1,77 @@
+use rand::prelude::IteratorRandom;
+use rand::rngs::StdRng;
+use rand::SeedableRng;
+
+use crate::simulation::replanning::routing::dijsktra::Dijkstra;
+use crate::simulation::replanning::routing::graph::ForwardBackwardGraph;
+
+pub type ForwardBackwardTravelTime = (u32, u32);
+
+const DEFAULT_NUMBER_OF_LANDMARKS: usize = 16;
+
+#[allow(dead_code)]
```

Comment:
Why dead code?

Reply:
the landmarks field is actually never used because the preprocessed data is in travel_times_to_all. However, I thought it might be good store the landmarks anyway.